src/content/docs/en/guides/media/cloudinary.mdx

```diff
@@ -6,11 +6,214 @@ stub: true
+
+```jsx
+---
+import { CldVideoPlayer } from 'astro-cloudinary';
+---
+<CldVideoPlayer
+  src="<Public ID>"
+  width="<Width>"
+  height="<Height>"
+/>
+```
+
+[Learn more about CldVideoPlayer](https://astro.cloudinary.dev/cldvideoplayer/basic-usage).
+
+## Uploading Files
+
+To add file uploading in your website or app's UI, you can use the CldUploadWidget, which will embed the [Cloudinary Upload Widget](https://cloudinary.com/documentation/upload_widget).
+
+For unsigned uploads, create an unsigned [Upload Preset](https://cloudinary.com/documentation/upload_presets#banner) and add:
+
+```jsx
+---
+import { CldUploadWidget } from 'astro-cloudinary';
+---
+<CldUploadWidget uploadPreset="<Upload Preset>">
+  <button>Upload</button>
+</CldUploadWidget>
+```
+
+For signed uploads, you can find [a guide and example](https://astro.cloudinary.dev/clduploadwidget/signed-uploads) on the Astro Cloudinary docs.
+
+[Learn more about CldUploadWidget](https://astro.cloudinary.dev/clduploadwidget/basic-usage).
+
+## Loading Images & Videos
+
+The Astro Content Layer can be used to load Cloudinary assets with the `cldAssetsLoader`.
+
+To load images or videos, add:
+
+```js
+import { defineCollection } from 'astro:content';
+import { cldAssetsLoader } from 'astro-cloudinary/loaders';
+
+export const collections = {
+  assets: defineCollection({
+    loader: cldAssetsLoader({
+      folder: '<Folder>' // Optional, without loads root directory
+    })
+  }),
+}
+```
+
+You can then use the [getCollection or getEntry functions](/en/guides/content-collections/#querying-collections) to select one or many images or videos from your collection.
+
+[Learn more about cldAssetsLoader](https://astro.cloudinary.dev/cldassetsloader/basic-usage).
+
+## Generating Social Media Cards
+
+Social Media Cards are useful for increasing engagement on social posts.
+
+To create a Social card, generate a Cloudinary URL and utilize Open Graph and Twitter meta tags to embed the image on your website:
+
+```jsx
+---
+import { getCldOgImageUrl } from 'astro-cloudinary/helpers';
+const ogImageUrl = getCldOgImageUrl({ src: '<Public ID>' });
+---
+<meta property="og:image" content={ogImageUrl} />
+<meta property="og:image:secure_url" content={ogImageUrl} />
+<meta property="og:image:width" content="1200" />
+<meta property="og:image:height" content="630" />
+<meta property="twitter:title" content="<Twitter Title>" />
+<meta property="twitter:card" content="summary_large_image" />
+<meta property="twitter:image" content={ogImageUrl} />
+```
+
+The `getCldOgImageUrl` helper utilizes the same API as the CldImage component, meaning you can use Cloudinary transformations to create dynamic, unique social cards for any of your pages.
+
+Find [Social Media Card templates](https://astro.cloudinary.dev/templates/social-media-cards) on the Astro Cloudinary docs.
```

Comment:
```suggestion
Find [Cloudinary Social Media Card templates](https://astro.cloudinary.dev/templates/social-media-cards) on the Cloudinary docs.
```
We can't exactly call them "Astro" here! :joy: I think in this context, it's fine just to say Cloudinary. What's important is that we make it clear these are external docs, to the site they're on, which is Astro Docs.

Reply:
oh i understand, that makes sense. my bad